### PR TITLE
pump/storage: Avoid big inputs that slow down the unit test

### PR DESCRIPTION
**Pull request by suzaku**
### What problem does this PR solve? 

With big inputs like "writing a 500MB binlog", the unit tests of
pump/storage can be very slow.

### What is changed and how it works?

1. Remove big inputs.
1. Batch valueLog writes

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

### CODE REVIEW

File: pump/storage/storage_test.go [hunk 1]
```diff
@@ -281,10 +276,15 @@ func populateBinlog(b Log, append *Append, prewriteValueSize int, binlogNum int3
 				binlog.Tp = pb.BinlogType_Prewrite
 				startTS := getTS()
 				binlog.StartTs = startTS
-				binlog.PrewriteValue = prewriteValue
+				if fixedValueSize {
+					binlog.PrewriteValue = prewriteValue
+				} else {
+					size := 1 + rand.Intn(len(prewriteValue)-1)
```
Comment (july2993) [inline pump/storage/storage_test.go]: you want `1 + rand.Intn(len(prewriteValue)` ?

File: pump/storage/storage_test.go [hunk 2]
```diff
@@ -257,17 +248,21 @@ func (as *AppendSuit) TestReadWriteGCTS(c *check.C) {
 
 // test helper to write binlogNum binlog to append
 func populateBinlog(b Log, append *Append, prewriteValueSize int, binlogNum int32) {
```
Comment (july2993) [inline pump/storage/storage_test.go]: add comment on how  's behavior  when `prewriteValueSize`  <= 0,  and seems no where use it as `prewriteValueSize > 0 ` now for `fixed stype` now?

File: pump/storage/storage_test.go (hunk 1 above)
Comment (suzaku) [reply pump/storage/storage_test.go]: It should be 1 less than the size because we want it to be `1 + [0,n-2] = [1, n-1]`.

**Pull request merged**